test/unit/visualizations/plugins/test_VisualizationsRegistry.py

```diff
@@ -89,11 +89,6 @@ def test_plugin_load( self ):
                     }
                 },
                 'not_a_vis1': {
-                    'config': {
-                        'vis1.xml': 'blerbler'
-                    },
-                },
-                'not_a_vis1': {
```

Comment:
Dunno, was like this from the very beginning of this file, the second dict entry happily shadowing the first one.